greedy_algorithms/jumpgame.cpp

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file
+ * @brief Implementation of an algorithm to solve the jumping game problem
```

Comment:
If possible, provide a Wikipedia link in [Markdown](https://www.markdownguide.org/) format.